### PR TITLE
Prevent unnecessary bandwidth from activated ACPs

### PR DESCRIPTION
**Pull request by dhrubabasu**
## Why this should be merged

Removes broadcasting activated ACPs in the handshake

## How this works

pretty straightforward

## How this was tested

CI

### CODE REVIEW

File: utils/constants/acps.go [hunk 1]
```diff
@@ -11,23 +11,23 @@ var (
...
-		31, // https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/31-enable-subnet-ownership-transfer.md
-		41, // https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/41-remove-pending-stakers.md
-		62, // https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/62-disable-addvalidatortx-and-adddelegatortx.md
+		23, // https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/23-p-chain-native-transfers/README.md
+		24, // https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/24-shanghai-eips/README.md
+		25, // https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/25-vm-application-errors/README.md
+		30, // https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/30-avalanche-warp-x-evm/README.md
+		31, // https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/31-enable-subnet-ownership-transfer/README.md
+		41, // https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/41-remove-pending-stakers/README.md
+		62, // https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/62-disable-addvalidatortx-and-adddelegatortx/README.md
```
Comment (StephenButtolph) [inline utils/constants/acps.go:31]: Since these have been included - should we just delete these?

Comment (dhrubabasu) [reply utils/constants/acps.go]: moved to an `ActivatedACPs` variable

**Review by StephenButtolph — APPROVED** [review]

File: RELEASES.md [hunk 2]
```diff
@@ -342,13 +342,13 @@ The plugin version is updated to `34` all plugins must update to be compatible.
...
-- [ACP-62](https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/62-disable-addvalidatortx-and-adddelegatortx.md) Disable AddValidatorTx and AddDelegatorTx
+- [ACP-23](https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/23-p-chain-native-transfers/README.md) P-Chain Native Transfers
+- [ACP-24](https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/24-shanghai-eips/README.md) Activate Shanghai EIPs on C-Chain
+- [ACP-25](https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/25-vm-application-errors/README.md) Virtual Machine Application Errors
+- [ACP-30](https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/30-avalanche-warp-x-evm/README.md) Integrate Avalanche Warp Messaging into the EVM
+- [ACP-31](https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/31-enable-subnet-ownership-transfer/README.md) Enable Subnet Ownership Transfer
+- [ACP-41](https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/41-remove-pending-stakers/README.md) Remove Pending Stakers
+- [ACP-62](https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/62-disable-addvalidatortx-and-adddelegatortx/README.md) Disable AddValidatorTx and AddDelegatorTx
```
Comment (joshua-kim) [inline RELEASES.md:351]: This is a good change but it feels like this should be in separate PR since it's technically unrelated IMO

Comment (dhrubabasu) [reply RELEASES.md:351]: fair point- i'm inclined to keep it in the same one out of laziness haha

File: utils/constants/acps.go [hunk 3]
```diff
@@ -6,28 +6,25 @@ package constants
...
+	)
+
 	// CurrentACPs is the set of ACPs that are currently, at the time of
 	// release, marked as implementable and not activated.
 	//
 	// See: https://github.com/orgs/avalanche-foundation/projects/1
-	CurrentACPs = set.Of[uint32](
```
[the remark this answers is not in the document]
Comment (dhrubabasu) [reply utils/constants/acps.go:13]: There's always going to be some lag since the ACPs will only be activated after the majority of the network chooses to activate it... not super sure how to go about this in AvalancheGo

File: RELEASES.md (hunk 2 above)
Comment (dhrubabasu) [reply RELEASES.md:351]: https://github.com/ava-labs/avalanchego/pull/3037

**Review by joshua-kim — APPROVED** [review]
